internal/overlord/servstate/handlers.go

```diff
@@ -84,6 +86,20 @@ func (m *ServiceManager) doStart(task *state.Task, tomb *tomb.Tomb) error {
 	cmd := exec.Command(args[0], args[1:]...)
 	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
 
+	// Start as another user if "user" specified in plan
+	if service.User != "" {
```

Comment:
I think user should be able to take a uid as well. Not all containers images will have a /etc/passwd

I'd follow https://docs.docker.com/engine/reference/run/#user

Reply:
Good call -- done. I've allowed numeric UID and GID as well. However, I've done them as two separate fields -- seeing this is structured YAML already, I don't think it makes sense to have to parse the `user[:group]` mini-format within a single field; might as well use separate fields (it also fits the files API better where `user` and `group` are two separate fields).

Reply:
I agree that in an API user and group should be separate. I can understand why they weren't for the CLI, but it makes more sense to be precise here.